.markdownlint.yml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+# Default state for all rules
+default: true
+
+# Line length : https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md013.md
+MD013:
+  line_length: 100
+
+# Inline HTML : https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md033.md
+MD033:
+  allowed_elements: [ img ]
```

Comment:
@volodya-lombrozo you can simply use Markdown image: https://stackoverflow.com/questions/41604263/how-do-i-display-local-image-in-markdown or maybe even remove the logo from the README

Reply:
@volodya-lombrozo there is yet another way: changing the size of the log in SVG. This is how we did it recently, with another logo (a similar problem): https://github.com/jcabi/jcabi-parent/pull/609/files Maybe you can do the same with our cactus SVG, which is here: https://github.com/objectionary/objectionary.github.io If you set its size to 64x64, we'll be able to use in all README files in all repos without problems.

Reply:
@yegor256 Now we are using a fixed size image. I removed this check